resources/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/kubernetes-resources.yaml

```diff
@@ -140,20 +140,4 @@ spec:
 {{- if .Values.defaultRules.additionalRuleLabels }}
 {{ toYaml .Values.defaultRules.additionalRuleLabels | indent 8 }}
 {{- end }}
-    - alert: CPUThrottlingHigh
```

Comment:
Also, can you explain the reason for this to be considered logspam? In what situations it is counter-productive?